robottelo/ui/locators.py

```diff
@@ -957,7 +957,8 @@ def __iter__(self):
     "host.vm_memory": (By.ID, "host_compute_attributes_memory"),
     "host.network_type": (
         By.ID, "host_compute_attributes_nics_attributes_0_type"),
-    "host.network": (By.ID, "host_compute_attributes_nics_attributes_0_type"),
+    "host.network": (
+        By.ID, "host_compute_attributes_nics_attributes_0_bridge"),
```

Comment:
I think it was a typo in automation code. As per above code snippet `host.network_type` and `host.network` were using same locator ID, which is not possible. So must be a typo.